spotinst/commons/common_set_credential_aws.go

```diff
@@ -0,0 +1,88 @@
+package commons
+
+import (
```

Comment:
change the file name to common_credentials_aws and use credentials wherever it is credential

Reply:
updated name to common_credentials_aws.go